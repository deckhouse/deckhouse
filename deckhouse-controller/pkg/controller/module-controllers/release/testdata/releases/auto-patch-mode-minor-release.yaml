---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: "90f0955ee984feab5c50611987008def"
  creationTimestamp: null
  generation: 1
  name: test
  resourceVersion: "406077723"
  uid: 94494a1a-868c-441f-a7a1-3c5e8ccf0957
spec:
  registry:
    ca: ""
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/deckhouse/modules
    scheme: HTTPS
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: parca-1.26.2
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: test
    uid: 94494a1a-868c-441f-a7a1-3c5e8ccf0957
spec:
  moduleName: parca
  version: 1.26.2
  weight: 900
status:
  approved: true
  phase: Deployed
  transitionTime: "2021-12-08T08:34:01Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: parca-1.27.0
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: test
    uid: 94494a1a-868c-441f-a7a1-3c5e8ccf0957
spec:
  moduleName: parca
  version: 1.27.0
  weight: 900
status:
  approved: true
  phase: Pending
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: parca
status:
  phase: Ready
