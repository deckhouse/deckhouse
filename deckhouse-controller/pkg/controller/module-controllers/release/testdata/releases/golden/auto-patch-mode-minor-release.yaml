---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
  name: parca-1.26.2
  resourceVersion: "1000"
spec:
  moduleName: parca
  version: 1.26.2
  weight: 900
status:
  approved: true
  message: ""
  phase: Deployed
  pullDuration: 0s
  size: 0
  transitionTime: "2021-12-08T08:34:01Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    modules.deckhouse.io/isUpdating: "false"
    modules.deckhouse.io/notified: "true"
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: ""
  name: parca-1.27.0
  resourceVersion: "1004"
spec:
  moduleName: parca
  version: 1.27.0
  weight: 900
status:
  approved: true
  message: 'Release is waiting for the ''modules.deckhouse.io/approved: "true"'' annotation'
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: null
